### ISSUE TITLE
request for compiled appimage

### ISSUE THREAD

**Issue by freddii** [zcot/conky-manager2#4]
Getting a ready compiled appimage would be really usefull

**Comment by probonopd** [comment]
Should maybe suggest over at https://github.com/teejee2008/conky-manager as well.

**Comment by freddii** [comment]
Requested it there too.

**Comment by zcot** [comment]
I don't know what it is, but I did a quick search and I do see some requirement in setting one up: https://github.com/AppImage/AppImageKit/wiki/Creating-AppImages

It's unfortunate that you have to gather binaries for different versioning, and by the end you are possibly required to do careful hand-tuning for different systems and versions(and now you end up having to develop for 7 different appimages? and users are having to wade through that too?). It looks similar to creating .deb packages except it's more steps and looks a lot more involved. For that matter I think there's an automated script in this source code that teejee originally wrote to create a .deb out of the build. -it's an unofficial "dirty" package, but the functionality is there.

On this note of easy installation, Flatpak might be a more straight-forward easier solution: http://docs.flatpak.org/en/latest/first-build.html

Personally, although it's a few steps to get it done, I don't see where building this package with the 3 or 4 commands is even that time consuming to matter: https://github.com/zcot/conky-manager2/blob/add-1.10-support/HOWTOBUILD.md

1. copy/paste the dependency package install line
2. clone the git repo or download the zip file
3. make and install

But maybe someone will come along and want to do this appimage thing. It's very easy to pull a fork in here to go build one, and even send back a pull request for it to be included in this source.

Really, the best solution is for the Master's of the Universe(Ubuntu Universe repository maintainers) to package it up as is, and then we can just install it using our preferred install method.

**Comment by probonopd** [comment]
> It's unfortunate that you have to gather binaries for different versioning, and by the end you are possibly required to do careful hand-tuning for different systems and versions(and now you end up having to develop for 7 different appimages?

No. You only need to make _one AppImage_ (for the oldest system that you are targeting, we recommend the oldest still-supported LTS release of Ubuntu which currently is trusty.)

From where in the documentation did you get this impression?

**Comment by zcot** [comment]
@probonopd I don't know anything about it. I just had done a quick search to even see what appimage is and saw documentation and a link to creating app images wiki(as linked above). That was as far as I looked since it laid out an advanced chain of events, without even reading the first points 3, 4, or 5.. -that was more involved than installing teejee's package.

That was as far as I got. Like I said, maybe someone else will want to do it.
